src/api/entities/MultiSig.ts

```diff
@@ -0,0 +1,95 @@
+import BigNumber from 'bignumber.js';
+
+import { MultiSigProposal } from '~/api/entities/MultiSigProposal';
+import { Account, Identity, PolymeshError } from '~/internal';
+import { ErrorCode, Signer } from '~/types';
+import { accountIdToString, identityIdToString, u64ToBigNumber } from '~/utils/conversion';
+
+interface MultiSigDetails {
+  signers: Signer[];
+  signaturesRequired: BigNumber;
+}
+
+/**
+ * Represents a MultiSig Account. MultiSig functions like an { @link Account } except submitted extrinsics need to be approved by a number of signers
+ */
+export class MultiSig extends Account {
```

Comment:
This class should override the `getIdentity` method using `multiSig.multiSigToIdentity` to fetch the DID

Reply:
Seems like I misunderstood what `multiSig.multiSigToIdentity` does